_configs/eval_ssm.yaml

```diff
@@ -68,6 +68,19 @@ infmax_models:
         max_nb_clusters: 10
         experiment_name: "My Experiment"
         visualise: False
+  - name: "GBIM"
+    class: "GBIM"
+    parameters:
+      rng_seed: 43
+      device: cpu
+      common:
+        nb_seeds: 5
```

Comment:
[3/3] Could you please use this field from `run` and put it here to `auto`. Then revise a loader for this model following example of `MultiNode2VecKMeansAuto`?